src/stampit.js

```diff
@@ -6,80 +6,77 @@
  * Copyright (c) 2013 Eric Elliott
  * http://opensource.org/licenses/MIT
  **/
-'use strict';
-var forEach = require('lodash/collection/forEach');
-var map = require('lodash/collection/map');
-var forOwn = require('lodash/object/forOwn');
-var isFunction = require('lodash/lang/isFunction');
-var isArray = Array.isArray;
-var isObject = require('lodash/lang/isObject');
-var create = Object.create;
-var slice = require('lodash/array/slice');
-var mixer = require('supermixer');
+import forEach from 'lodash/collection/forEach';
+import forOwn from 'lodash/object/forOwn';
+import isFunction from 'lodash/lang/isFunction';
+import isObject from 'lodash/lang/isObject';
+import map from 'lodash/collection/map';
+import * as mixer from 'supermixer';
+
+const create = Object.create;
+const isArray = Array.isArray;
 
 /* jshint -W024 */
 
 // Avoiding JSHist W003 violations.
-var stampit;
+let stampit;
 
-function extractFunctions(arg) {
-  if (isFunction(arg)) {
-    return map(slice(arguments), function (fn) {
+function extractFunctions(...args) {
+  if (isFunction(args[0])) {
+    return map(args, fn => {
       if (isFunction(fn)) {
         return fn;
       }
     });
-  } else if (isObject(arg)) {
-    var arr = [];
-    forEach(slice(arguments), function (obj) {
-      forOwn(obj, function (fn) {
+  } else if (isObject(args[0])) {
+    let arr = [];
+    forEach(args, obj => {
+      forOwn(obj, fn => {
         arr.push(fn);
       });
     });
     return arr;
-  } else if (isArray(arg)) {
-    return slice(arg);
+  } else if (isArray(args[0])) {
+    return args[0];
   }
   return [];
 }
 
-function addMethods(fixed, methods) {
-  var args = [fixed.methods].concat(methods);
-  mixer.mixinFunctions.apply(null, args);
+function addMethods(fixed, ...methods) {
```

Comment:
The function can be shortened to a single line now, if I remember ES6 correctly .